RecoTracker/LST/plugins/LSTOutputConverter.cc

```diff
@@ -25,15 +25,15 @@
 #include "TrackingTools/Records/interface/TrackingComponentsRecord.h"
 #include "TrackingTools/TrajectoryState/interface/TrajectoryStateTransform.h"
 
-class LSTOutputConverter : public edm::global::EDProducer<> {
+class LSTOutputConverter : public edm::stream::EDProducer<> {
```

Comment:
On this change, I am not sure how to make this work with `StreamCache`, as the constructor of `SeedCreator` requires the edm::Event and the edm::EventSetup to get the parameters and ES product, which are not available when one constructs the object in the `beginStream`. I ended up moving the module to edm::stream::EDProducer, which is an acceptable solution, according to Matti.
If you see something that I have missed, I'd be up for implementing it.